GitHubDashboard/Controllers/QueryController.cs

```diff
@@ -54,7 +54,16 @@ public QueryController(GitHubClient gitHubClient)
 
         private async Task<IReadOnlyList<Issue>> GetIssuesAsync(string owner, string repository, string milestone, string labels)
         {
-            if (string.IsNullOrWhiteSpace(milestone) || milestone == "any" || milestone == "undefined" || milestone == "*")
+            // "undefined" means that milestone was not set on the incoming URL.  Something like this:
```

Comment:
"incoming URL" is a bit confusing in this case, since there are two:

1. From this (the json microservice) code's point of view, the incoming URL is the MVC one where the label will be `/` separated.
2. The URL routing exposed by the Angular app.

Can you clarify the comment to say that the Angular app takes urls like "http://<host>/count/nuget/home?label=VS1ES", and calls the service with a URL like "http://host/count/nuget/home/undefined/VS1ES"?